scripts/apps/ingest/index.ts

```diff
@@ -135,7 +136,10 @@ angular.module('superdesk.apps.ingest', [
                 }],
                 filters: [{action: 'list', type: 'externalsource'}],
                 privileges: {fetch: 1},
-                additionalCondition: ['desks', function(desks) {
+                additionalCondition: ['desks', 'item', function(desks, item) {
+                    if (item._fetchable !== undefined) {
+                        return isMediaEditable(item) && !_.isNil(desks.getCurrentDeskId());
```

Comment:
I think you can change this whole function to be just this line, it checks `appConfig` features if `_fetchable` is not set on item

Reply:
so it would be:
```ts
additionalCondition: ['desks', 'item', (desks, item) => isMediaEditable(item) && !_.isNil(desks.getCurrentDeskId()]
```

Reply:
I agree. I'd also add a comment that desk is checked in order to prevent fetching to personal space. It's clear now, but might not be in the future.

Reply:
I kept that comment.